docs/modules/plugins/pages/plugin-sauce-labs.adoc

```diff
@@ -110,3 +110,14 @@ host patterns to avoid sending of status information by SauceConnect to SauceLab
 a tunnel via proxy used by tests.
 
 |===
+
+.Configuration snippet to run tests in Chrome browser on Sauce Labs US West data center
+[source,json,subs=attributes+, source-highlighter+]
```

Comment:
```suggestion
[source,properties,subs=attributes+, source-highlighter+]
```

Reply:
and why is `subs=attributes+, source-highlighter+` needed?

Reply:
copy paste :)